Sources/Commands/SwiftPackageTool.swift

```diff
@@ -341,6 +344,32 @@ extension SwiftPackageTool {
         }
     }
     
+    struct UpdateTemplate: SwiftCommand {
+        static let configuration = CommandConfiguration(
+            abstract: "Update template from remote")
+        
+        @OptionGroup(_hiddenFromHelp: true)
+        var swiftOptions: SwiftToolOptions
+        
+        @Argument(help: "Template to update")
+        var templateName: String
+        
+        func run(_ swiftTool: SwiftTool) throws {
+            guard let configPath = try swiftTool.getConfigPath() else {
+                throw InternalError("Could not find config path")
+            }
+            
+            guard localFileSystem.exists(configPath.appending(components: "templates", "new-package", templateName)) else {
+                throw InternalError("Could not find \(templateName)")
```

Comment:
this should not be an InternalError, as its a user error (bad input). use StringError instead.